### PR TITLE
Fix typo that prevents calling `ExactState.quantum_geometric_tensor`

### PR DESCRIPTION
**Pull request by femtobit**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 ### Bug Fixes
 * The gradient obtained with `VarState.expect_and_grad` for models with real-parameters was off by a factor of $ 1/2 $ from the correct value. This has now been corrected. As a consequence, the correct gradient for real-parameter models is equal to the old times 2. If your model had real parameters you might need to change the learning rate and halve it. [#1069](https://github.com/netket/netket/pull/1069)
 * Support for coloured edges in `nk.graph.Grid`, removed in [#724](https://github.com/netket/netket/pull/724), is now restored. [#1074](https://github.com/netket/netket/pull/1074)
+* Fixed bug that prevented calling `.quantum_geometric_tensor` on `netket.vqs.ExactState`. [Insert link]
```
Comment (femtobit) [inline CHANGELOG.md:22]:
```suggestion
* Fixed bug that prevented calling `.quantum_geometric_tensor` on `netket.vqs.ExactState`. [#1108](https://github.com/netket/netket/pull/1108)
```

File: test/variational/test_exact_state.py [hunk 2]
```diff
@@ -89,6 +89,19 @@ def _f(x, y):
...
+def test_basic_methods(vstate):
+    key1, key2 = jax.random.split(nk.jax.PRNGKey())
+
+    s = vstate.hilbert.random_state(key1)
+    assert np.shape(vstate.log_value(s)) == ()
+
+    s = vstate.hilbert.random_state(key2, size=2)
+    assert np.shape(vstate.log_value(s)) == (2,)
+
+    _ = vstate.quantum_geometric_tensor()
```
Comment (PhilipVinc) [inline test/variational/test_exact_state.py:102]: Can we check anything of the returned object?

**Review by PhilipVinc — APPROVED** [review]

File: test/variational/test_exact_state.py (hunk 2 above)
Comment (femtobit) [reply test/variational/test_exact_state.py:102]: Maybe something like `hasattr(qgt, "solve")`? The QGT classes don't have a common base type.

Comment (PhilipVinc) [reply test/variational/test_exact_state.py:102]: nk.optimizer.LinearOperator I think

**Pull request merged**